docs/ref/contrib/auth.txt

```diff
@@ -30,10 +30,7 @@ Fields
 
         The ``max_length`` should be sufficient for many use cases. If you need
         a longer length, please use a :ref:`custom user model
-        <specifying-custom-user-model>`. If you use MySQL with the ``utf8mb4``
-        encoding (recommended for proper Unicode support), specify at most
-        ``max_length=191`` because MySQL can only create unique indexes with
-        191 characters in that case by default.
+        <specifying-custom-user-model>`.
```

Comment:
This looks to [still be true for InnoDB](https://dev.mysql.com/doc/refman/8.4/en/charset-unicode-conversion.html) for tables that use [COMPACT](https://dev.mysql.com/doc/refman/8.4/en/glossary.html#glos_compact_row_format) or [REDUNDANT](https://dev.mysql.com/doc/refman/8.4/en/glossary.html#glos_redundant_row_format) row format.

This was documented here as this limitation was the reason the max length was not increased to 254 characters (see 780bddf75b93784470a2e352ed44ee35a751d667 and https://docs.djangoproject.com/en/5.1/releases/1.10/#abstractuser-username-max-length-increased-to-150)

I _think_ it is ok to remove this note as the context here is around customizing the user model and folks should bare in mind database limitations when making such a customization.
I also don't think we need to revisit the decision for the max length to be 150 here.